### PR TITLE
Closes issue #17 Fixing sporadic restore failures for blueprint

### PR DESCRIPTION
**Pull request by pritishmishra**
Description: Restore of blueprint used to read restore.output.json
and failed due to sporadic non-existence of this file. Since, this
file is of no consequence to restore operation and was present to
maintain generic functionality, now, not reading this file for
restore operation.

### CODE REVIEW

**Review by kamath-prasad — APPROVED** [review]
lgtm , will merge after another approval.

**Review by sauravmndl-zz — APPROVED** [review]
LGTM

**Pull request merged**
